tidb-cloud/tidb-cloud-billing.md

```diff
@@ -32,10 +32,10 @@ See the detailed pricing for each available Alibaba Cloud region below.
 | Resource | Singapore |
 |----------|-----------|
 | Compute (per RCU/month) | $0.24 |
-| Row storage (per GiB/month) | $0.24 |
-| Column storage (per GiB/month) | $0.06 |
-| Row storage with enhanced encryption at rest (per GiB/month) | $0.36 |
-| Column storage with enhanced encryption at rest (per GiB/month) | $0.09 |
+| Row-based storage (per GiB/month) | $0.24 |
+| Columnar storage (per GiB/month) | $0.06 |
+| Row-based storage with dual-layer encryption (per GiB/month) | $0.36 |
+| Columnar storage with dual-layer encryption (per GiB/month) | $0.09 |
 
```

Comment:
PS: the preceding changes are to align with https://github.com/pingcap/docs/pull/21516